pycqed/analysis/measurement_analysis.py

```diff
@@ -681,6 +682,11 @@ def get_naming_and_values(self, new_sweep_points=None):
             self.sweep_points_from_file = self.sweep_points
             self.sweep_points = new_sweep_points
 
+        try:
+            self.exp_metadata = h5d.read_dict_from_hdf5(
+                {}, self.data_file['Experimental Data']['Experimental Metadata'])
+        except KeyError:
+            self.exp_metadata = {}
```

Comment:
Will this fix the issue discussed in my pull request of receiving the `[]` instead of `{}` for the experimental metadata?

Reply:
No, this is in analysis_v1, your issue was in v2.